src/test/scala/org/broadinstitute/dsde/rawls/webservice/WorkspaceApiServiceSpec.scala

```diff
@@ -369,10 +369,11 @@ class WorkspaceApiServiceSpec extends ApiServiceSpec {
           status
         }
       }
+    //this endpoint is only temporarily returning MethodNotAllowed. When GAWB-422 is complete, it should return NotFound
     Delete(s"/workspaces/${testData.workspace.namespace}/${testData.workspace.name}x/entities/${testData.sample2.entityType}/${testData.sample2.name}") ~>
       sealRoute(services.entityRoutes) ~>
       check {
-        assertResult(StatusCodes.NotFound) {
+        assertResult(StatusCodes.MethodNotAllowed) {
```

Comment:
Didn't disable this test because the rest of it is still relevant. This will break once we fix entity-delete, but I left a note so we'll know immediately what to change when it does break :)